### DISCUSSION TITLE
The simulated results are not coinciding with the results of Triaxial compression test in tutorial manual,Chapter 4.

### DISCUSSION THREAD

**Discussion by zhangfengeo** [Anura3D/Anura3D_OpenSource#47] [Q&A]
Hi,I build a  model of Triaxial Compression Test and set parameters according to the tutorial manual in chapter 4, but the simulation result are different with the tutorial manual's.For example,the deviatoric stress range I got is 0~50kpa, whereas the result in manua is 0~150kpa,and the mean effect stress(-67~-50kpa I got) and volumetric strain(-2.5e-3~0 I got) are also different with the manual. Can someone help me to verify the result again? The attach files are the GOM file and CPS file I got.
[GOM and CPS file.zip](https://github.com/Anura3D/Anura3D_OpenSource/files/10047429/GOM.and.CPS.file.zip)
![picture01](https://user-images.githubusercontent.com/103553910/202860828-fa8d6763-f999-4a32-886a-a08285efa2c9.PNG)
![picture2](https://user-images.githubusercontent.com/103553910/202860831-e85baf3e-3372-4c44-a29b-5fedc82aa160.PNG)

**Comment by WaveHello** [comment]
Hello,

There are two errors in the tutorial manual that may be leading to confusion. The first is a typo in the "Applying prescribed velocity" section and the second is an error associated with the x-axes (Dev strain) of Figure 4.19 and Figure 4.20 at the end of the Triaxial Example.

1) The 3rd and 4th step of the  "Applying prescribed velocity" section says to prescribe a velocity of -0.001 m/s it should be -0.01 m/s. For future reference, the correct value is written in the introduction paragraph of the "Applying prescribed velocity" section and near the beginning of the Triaxial example in section 4.1.

2) The x-axes of Figure 4.19 and Figure 4.20 should be on the order of 1% instead of 0.1%. Therefore, the range of the axes should be 0% to 6%.

I've attached an updated version of the GOM file with the correct prescribed velocities. You can use the attached file or can go into the GOM file, scroll about halfway down until you see the "$$PRESCRIBED_NODAL_VELOCITY_SURFACE" header, and change all of the -0.001 values to -0.01. You could also specify the value in GiD and regenerate the .A3D folder.

Please let me know if this fixes the issue.
[Updated_GOM_File.zip](https://github.com/Anura3D/Anura3D_OpenSource/files/10484441/Updated_GOM_File.zip)

**Reply by DanBrox** [reply]
Is it possible to provide an explanation for why the solid displacements of the material points are 0 at all time steps even though there is an applied axial strain throughout the triaxial test?  
![TriaxialTestExample](https://github.com/user-attachments/assets/90767b29-b83e-4444-b7d7-866734008f4d)
